src/HDInsight/HDInsight/ManagementCommands/NewAzureHDInsightClusterConfigCommand.cs

```diff
@@ -148,6 +148,13 @@ public Guid AadTenantId
             set { _config.AADTenantId = value; }
         }
 
+        [Parameter(HelpMessage = "Gets or sets the minimal supported TLS version.")]
```

Comment:
The help doc for this cmdlet needs to be regenerated

Reply:
> The help doc for this cmdlet needs to be regenerated

Updated